test/src/tests-forwarders.js

```diff
@@ -2179,18 +2428,26 @@ describe('forwarders', function() {
         window.mParticle.config.kitConfigs.push(config1);
 
         mParticle.init(apiKey, window.mParticle.config);
-
+        waitForCondition(() => {
+            return (
+                window.mParticle.getInstance()._Store.identityCallInFlight === false
+            );
+        })
+        .then(() => {
         window.MockForwarder1.instance.should.have.property(
             'onUserIdentifiedCalled',
             true
         );
 
         done();
     });
+    });
 
-    it('should queue forwarder stats reporting and send after 5 seconds if batching feature is true', function(done) {
-        const clock = sinon.useFakeTimers();
-
+    // This test sends to forwarding batch endpoint so the fact that it is failing is not a big deal as we should probably just delete that code.
```

Comment:
Make this a ticket.

Reply:
done - https://mparticle-eng.atlassian.net/browse/SQDSDKS-6850